text/0000-unify-katello-ui.md

```diff
@@ -0,0 +1,78 @@
+- Feature Name: Unify Katello UI
+- Start Date: 2016-08-26
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The goal of this feature is to combine [Katello](github.com/katello/katello) and [Bastion](github.com/katello/bastion)
+into a single UI project.
+
+# Motivation
+[motivation]: #motivation
+
+Currently when working on the UI in Katello a developer has to oftentimes make changes in both the Katello and Bastion
+projects in order to complete a feature.  Once Katello begins to make more use of the components from
+[angular-patternfly](https://github.com/patternfly/angular-patternfly) this problem will only become worse as there will
+be three potential UI repositories to alter in order to complete new features.
+
+The expected outcome of this feature is to reduce the complexity of working on UI code in Katello by combining projects.
+
+# Detailed design
+[design]: #detailed-design
+
+## Option 1:  Move Bastion code back into Katello
+
+Eliminate Bastion entirely and move everything in Bastion back into Katello.
+
+### Pros
+- One less repository to manage
+- Would no longer require the release of a Bastion gem to update UI code in Katello
+- UI changes would be easier because they would only be made in one repository/project
+
+### Cons
+- Decreases separation of concerns, the UI would be bundled with the API code (although you could count this as a pro
```

Comment:
I meant the entire test suite of checks we run on Katello today. Not that PRs would get slower, that one con to having the different concerns together is that pure JS changes require the Ruby tests to run making them slower to iterate.